crates/next-dev/benches/bundlers/turbopack/mod.rs

```diff
@@ -49,7 +49,7 @@ impl Bundler for Turbopack {
         npm::install(
             install_dir,
             &[
-                NpmPackage::new("next", "13.0.8-canary.2"),
+                NpmPackage::new("next", "13.1.7-canary.2"),
```

Comment:
This enables the next.js router for the benchmarks, which causes a slowdown

Reply:
Would be nice if we could declare this in some central place somehow